.github/workflows/create_release.yml

```diff
@@ -49,13 +49,16 @@ jobs:
           BUILD_DIR=${GITHUB_WORKSPACE}/build
           VM_GO_VERSION=$(cat go.mod | grep mx-chain-vm-go | sort -n | tail -n -1| awk -F '/' '{print$3}'| sed 's/ /@/g')
           VM_GO_DIR=${GOPATH}/pkg/mod/github.com/multiversx/${VM_GO_VERSION}
+          VM_14_VERSION=$(cat go.mod | grep mx-chain-vm-v | sort -n | tail -n -1| awk -F '/' '{print$3}'| sed 's/ /@/g')
```

Comment:
Here, we have `14` hardcoded in variable name, but expression is somehow more generic (uses prefix "v", does not mention exact version). We could have had them at the same level of "hardcodedness".

However, all good as it is, legacy VM version won't change etc.

Reply:
updated